### PR TITLE
Accessibility Link-Error Link Functionality Issue on Registration Date Page

### PR DESCRIPTION
**Pull request by olakunlekuye2**
Fixed the error link on the “Enter the group’s registration date” page in the Report Pillar 2 top-up taxes service that did not function as expected.

### CODE REVIEW

File: app/views/ViewUtils.scala [hunk 1]
```diff
@@ -64,10 +83,4 @@ object ViewUtils {
     val formattedTime = zonedTime.format(formatter)
     formattedTime
   }
-
-  def hideForScreenReader(visualKey: String, screenReaderKey: Option[String]): Html =
```
Comment (JamesMMiller) [inline app/views/ViewUtils.scala:68]: So this was unused?

Comment (olakunlekuye2) [reply app/views/ViewUtils.scala:68]: Yes and I thought it was forgotten , but happy to leave be

Comment (olakunlekuye2) [reply app/views/ViewUtils.scala:68]: Okay I'll add it back

**Comment by JamesMMiller** [comment]
One general comment, why aren't we doing this same thing on `GroupAccountingPeriodView.scala.html`? Does it already work as expected there or do we have to implement this same errorKey1 method?

File: app/views/ViewUtils.scala [hunk 2]
```diff
@@ -37,6 +36,26 @@ object ViewUtils {
   def errorPrefix(form: Form[_])(implicit messages: Messages): String =
     if (form.hasErrors || form.hasGlobalErrors) messages("error.browser.title.prefix") else ""
 
+  def errorKey1(form: Form[_], fieldKey: String): String = {
```
Comment (JamesMMiller) [inline app/views/ViewUtils.scala:39]: Could we have a different name on this? like localDateErrorKey or something more descripive?

Comment (olakunlekuye2) [reply app/views/ViewUtils.scala:39]: Okay

**Review by JamesMMiller — APPROVED** [review]

**Pull request merged**